src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -594,9 +594,10 @@ private void AddSnippetsToResults()
             {
                 string snippetId, snippetText;
                 if (_resultToSnippetIdDictionary.TryGetValue(result, out snippetId) &&
-                    _snippetIdToSnippetTextDictionary.TryGetValue(snippetId, out snippetText))
+                    _snippetIdToSnippetTextDictionary.TryGetValue(snippetId, out snippetText) &&
+                    result.Locations[0]?.PhysicalLocation?.Region != null)
```

Comment:
Shouldn't in be `result.Locations?[0]?.`?

Reply:
Yep thanks